cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -135,6 +139,29 @@ def csv_to_table(file_operations: list):
         print("    created table:" + file["table_name"])  # functional
 
 
+@register_pipeline_stage("backup_files")
+def backup_files(file_list: List[str], backup_directory: str):
+    """
+    Backup a list of files on the local or hdfs file system to a hdfs backup directory
+    """
+    storage_dir = backup_directory + "/" + datetime.now().strftime("%Y%m%d_%H%M%S")
+    if not isdir(storage_dir):
+        if create_dir(storage_dir):
+            print(f"    created dir: {storage_dir}")  # functional
+        else:
+            print(f"    failed to create dir: {storage_dir}")  # functional
```

Comment:
Minor point, but we might want to raise an exception here? As we probably can't write the files if the directory can't be created